smoke-test.sh

```diff
@@ -259,17 +253,6 @@ function runAllTests() {
     fi
 }
 
-function runWebTests() {
-    doCommand C# web "$@" new restore run
-    doCommand C# mvc "$@" new restore run
-    doCommand C# webapi "$@" new restore run
-    doCommand C# razor "$@" new restore run
-
-    doCommand F# web "$@" new restore run
-    doCommand F# mvc "$@" new restore run
-    doCommand F# webapi "$@" new restore run
-}
-
 function resetCaches() {
```

Comment:
Are we not running web tests anymore?  If you're removing this function, you should remove calls to it as well.

Reply:
We actually a duplicate function, this just removes one of them.